src/lib/helpers/handleFileTracking.js

```diff
@@ -22,15 +22,13 @@ const handleFileTracking = ({ functionsPath, publishPath }) => {
     const trackingFile = readFileSync(trackingFilePath, 'utf8')
     const [trackedFunctions, trackedPublish] = trackingFile.split(TRACKING_FILE_SEPARATOR)
     const cleanConfiguredFiles = (trackedFiles, dirPath) => {
-      trackedFiles.forEach((file) => {
-        const filePath = join(dirPath, file.trim('\r'))
-        if (file !== '') {
-          if (process.platform === 'win32') {
-            rmdirSync(filePath, { recursive: true })
-          }
+      trackedFiles
+        .map((file) => file.trim())
+        .filter(Boolean)
```

Comment:
is this effectively the same thing as `if (file !== '')` ?

Reply:
In the original code `if (file !== '')` is the original non trimmed value (`trim` doesn't mutate the string), so the check should be `if (file.trim() !== '')`.
`filter(Boolean)` operates on the trimmed value.